src/Output/Output_DumpData.cpp

```diff
@@ -137,8 +137,10 @@ void Output_DumpData( const int Stage )
       sprintf( FileName_PS, "PowerSpec_%06d", DumpID );
 
 #  ifdef PARTICLE
-   if ( OPT__OUTPUT_PAR_TEXT )
+   if ( OPT__OUTPUT_MODE==1 )
       sprintf( FileName_Particle, "Particle_%06d", DumpID );
+   if ( OPT__OUTPUT_MODE==2 )
+      sprintf( FileName_Particle, "Particle_%06d.cbin", DumpID );
```

Comment:
- It should be `OPT__OUTPUT_PAR_MODE` instead of `OPT__OUTPUT_MODE`.
- How about using `"Particle_%06d.txt"` to distinguish it from `"Particle_%06d.cbin"`? Note that it will break the compatibility with some old gnuplot plotting scripts (e.g., `example/test_problem/Hydro/ClusterMerger/gnuplot_script/plot__particle_single-cluster.gpt` and `example/test_problem/Hydro/Plummer/plot_script/plot_particles_*.gpt`).
- Add one space before and after `==` to conform to the GAMER format.